src/api/c/graphics_common.cpp

```diff
@@ -366,6 +424,24 @@ forge::VectorField* ForgeManager::getVectorField(forge::Chart* chart, int nPoint
     return mVcfMap[keypair];
 }
 
+bool ForgeManager::getChartAxesOverride(forge::Chart* chart)
+{
+    ChartAxesOverrideIter iter = mChartAxesOverrideMap.find(chart);
+    if (iter == mChartAxesOverrideMap.end()) {
+        AF_ERROR("Chart Not Found!", AF_ERR_ARG);
```

Comment:
Yeah you are right, I'll change that.
In any case, it should never execute that statement.